log/examples.md

```diff
@@ -152,4 +152,41 @@ func loggingDemo(ctx context.Context) {
 
 ```
 
+### Logging in different formats
+
+There is the JSON format, you can set it by adding `WithConfig(log.JSONFormatter{})`. It will log in the following format:
+```json
```

Comment:
Tag it as `js` so comments aren't treated as errors.

Reply:
added